README.md

```diff
@@ -512,6 +516,208 @@ get '/', http_codes: [
+
+### Default Lookup Keys
+
+The following lookup keys are all within the scope given by `i18n_scope`.
+
+#### API Info
+
+The default lookup key of a field in API info is `info.<field>`, i.e.:
+
+* `info.title` for `:title` field.
+* `info.desc` or `info.description` for `:description` field.
+* `info.contact` for `:contact` field.
+* `info.license` for `:license` field.
+* `info.license_url` for `:license_url` field.
+* `info.terms_of_service_url` for `:terms_of_service_url` field.
+
+#### Namespaces/Resources Description
+
+Grape-swagger will give each root namespace a default description, such as "*Operations about
+users*", where `users` is a namespace (pluralized). But you can change it by provide your own
+message under key `<namespace>.desc` or `<namespace>.description` in your locale file. And the
+namespace itself is available as a variable for interpolation.
+
+### Endpoints Description and Detail/Notes
+
+An endpoint's default lookup key is in format of `<namespace>.<path>.<http_method>`. And if there
+are nested namespaces or multi-level path, all parts will be join by `.`. For example for this
+endpoint:
+
+```ruby
+namespace :users do
+  route_params :id do
+    get :'password/strength' do
+    end
+  end
+end
+```
+
+It's corresponding lookup key will be `users.:id.password.strength.get`, let's say that this is
+the key of this endpoint.
+
+So an endpoint's description message can be given under `<endpoint_key>.desc` or
+`<endpoint_key>.description`. And use `<endpoint_key>.detail` or `<endpoint_key>.notes` for the
+message of its further details.
+
+#### Params Description
+
+The first default key of endpoint parameter's description translation is
+`<endpoint_key>.params.<param_name>`. But considering that some endpoints could usually share a
+same parameter, it will be a little annoyed to duplicate its description message everywhere. So if
+no translation found in the first default key, grape-swagger will try to find all its parent keys.
+Take above endpoint for example, it may have a parameter named `:id`, then the lookup keys are:
+
+1. `users.:id.password.strength.get.params.id`
+1. `users.:id.password.strength.params.id`
+1. `users.:id.password.params.id`
+1. `users.:id.params.id`
+1. `users.params.id`
+1. `params.id`
+
+#### Models/Entities Description
+
+A model class' lookup key is by default its class name, without module part, and underscored. Each
+property's key is then `entities.<class_key>.<property_name>`. When not found, grape-swagger will
+try to check the its ancestors, up to a class whose name is `Entity`.
+
+Say if there is model class `User` inherits `Grape::Entity`, and another model `AdminUser` inherits
+`User`, to translate a property named `:email` of `AdminUser`, the lookup keys are:
+
+1. `entities.admin_user.email`
+1. `entities.user.email`
+1. `entities.default.email`
+
+#### All in All
+
+Here is a piece of example translation messages.
```

Comment:
This is not great English :) Change "All in All" to just "Example" and you don't need the "here ..." part.